src/components/Table/_table.scss

```diff
@@ -10,6 +11,10 @@ table.#{$prefix}--side-nav--data-table {
   table-layout: fixed;
 }
 
+.#{$iot-prefix}--table__cell--truncate {
+  max-width: 0;
+}
+
```

Comment:
Should this be moved to the `TableCellRenderer/_table-cell-renderer.scss`?

Reply:
Possibly, to me it isn't obvious where that style belongs. It relates to table cell rendering, but the style is actually for the parent  <TableCell> component and the class is set in the `TableBodyRow.jsx`.